src/hooks/useLanguageServer.ts

```diff
@@ -87,12 +88,21 @@ export default function useLanguageServer() {
     });
   };
 
+  const debouncedServerRestart = useMemo(
+      () => debounce(restartServer, 150),
+      [languageServer]
+  )
+
   useEffect(() => {
     if (languageServer) {
       languageServer.updateCodeGetter(getCode);
     }
   }, [project.project.accounts]);
 
+  // TODO: Disable this, once the cadence language server package is updated
+  useEffect(debouncedServerRestart, [project.project.accounts, project.active]);
```

Comment:
Just for the sake of learning, how does restarting the server on editor change mitigate the bug reported? :)

Reply:
It's not directly connected to #251 ,but current `staging` branch is using newer and a bit broken build of language server. So this PR reverts it back to older version, which requires LS to be restarted when contracts on account is changed :)